record-two/record.css

```diff
@@ -0,0 +1,3 @@
+.padding-top-10 {
```

Comment:
Looks like it's time to move this class to the common stylesheet. I've seen this used in recordedit(?) as well and this is more a utility-type class anyway.